lib/checkout.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'order'
+require_relative 'till'
+
+class Checkout
+
+  include Till
+
+  attr_reader :order
+
+  def initialize order
+    @order = order
+  end
+
+  def no_dishes_ordered?
+    @order.dishes.empty?
+  end
+
+  def send_msg
+    if no_dishes_ordered?
+      fail 'No dishes have been added to the order'
+    else
+      require_relative 'twilio.rb'
```

Comment:
Interesting inline use of require... I have never seen that before. It's better to have a wrapper class for a gem such as twilio (think of the wrapper class as a phone, and the twilio gem is the antennae.)